hi.js

```diff
@@ -1,821 +1,821 @@
+settings.client.angleBackend ऐंगल ग्राफ़िक्स बैकेंड
+settings.client.colorProfile रंग प्रोफ़ाइल
+settings.dev.header डेवलपर
+settings.dev.forceUpdate बलपूर्वक श्रृंगार बदलें
+settings.dev.forceRegen हमेशा खिलाड़ी को पुनर्जीवित करें
+settings.dev.extraBan बैन की अतिरिक्त जानकारी
+settings.chat.header चैट
+settings.chat.chat चैट बॉक्स दिखाएँ
+settings.chat.chat.all हमेशा
+settings.chat.chat.focus फोकस करने पर
+settings.chat.messages खिलाडियों के सन्देश दिखाएं
+settings.chat.unboxings अनबॉक्सिंग दिखाएं
+settings.chat.chatOp चैट अपारदर्शिता
+settings.chat.chatBGOp चैट BG अपारदर्शिता
+settings.chat.chatHeight चैट ऊंचाई
+settings.chat.chatTimer चैट टाइमर (s)
+settings.chat.chatTextOutline चैट टेक्स्ट का बॉर्डर
+settings.chat.chatFilter अपशब्दों को हटाएँ
+settings.quality.header पर्फोर्मेन्स
+settings.quality.updRate फ़्रेम कैप
+settings.quality.rendDst रेंडर दूरी
+settings.quality.res रेसोलुशन
+settings.quality.part कण
+settings.quality.mapDet नक्शा विवरण
+settings.quality.noTex कोई टेक्सचर नहीं
+settings.quality.partdist कण दूरी
+settings.quality.renderdist रेंडर दूरी
+settings.quality.reflection प्रतिबिंब की गुणवत्ता
+settings.quality.antiAlias एंटीअलियासिंग
+settings.quality.shadows छाया
+settings.quality.softShad नरम छाया
+settings.quality.highResShad उच्च रेसोलुशन छाया
+settings.quality.shadowsDynamic डायनामिक छाया
+settings.quality.trails बुलेट ट्रेल्स
+settings.quality.yourTrails अपने ट्रेल्स
+settings.quality.mflash मज़ल फ्लैश
+settings.quality.flap स्नाइपर फ्लैप
+settings.quality.lowSpec कम स्पेसिफ़िकेशन
+settings.quality.ambient एम्बियेन्ट शेडींग
+settings.quality.ambientOld पुरानी शेडींग (जब संभव हो)
+settings.quality.textureAnim टेक्सचर एनीमेशन
+settings.quality.objectAnim वस्तु एनिमेशन
+settings.quality.screenShake स्क्रीन शेक
+settings.quality.weaponShine हथियार की चमक
+settings.quality.rendGamma गामा
+settings.quality.lighting प्रकाशन
+settings.quality.showExplo विस्फोट दिखाएँ
+settings.quality.aspectRatio आस्पेक्ट रेशियो
+settings.post.header पोस्ट प्रोसेसिंग
+settings.post.post पोस्ट प्रोसेसिंग
+settings.post.bloom ब्लूम
+settings.post.bloomThreshold ब्लूम थ्रेशोल्ड
+settings.post.bloomStrength ब्लूम ताकत
+settings.post.bloomRadius ब्लूम रेडियस
+settings.post.ssao SSAO
+settings.post.ssaoRadius SSAO रेडियस
+settings.interface.hitInd निशान लगने पर संकेत दिखाएँ
+settings.interface.hitColor निशान संकेतों का रंग
+settings.interface.dmg निशान लगने पर डैमेज दिखाएँ
+settings.interface.dmgCol डैमेज संकेतों का रंग
+settings.interface.critCol घातक डैमेज संकेतों का रंग
+settings.interface.header इंटरफ़ेस
+settings.interface.scale UI का माप
+settings.interface.ui UI दिखाएँ
+settings.interface.scoreboard पुराने स्कोरबोर्ड का उपयोग करें
+settings.interface.oldBrowser पुराने ब्राउज़र का उपयोग करें
+settings.interface.rarityanim दुर्लभता एनिमेशन असमर्थ करें
+settings.interface.showPfp प्रोफ़ाइल चित्र दिखाएँ
+settings.interface.showTradeMsg व्यापार के संदेश दिखाएँ
+settings.interface.hideNonTrade ट्रेड न होने वाली स्किन्स को छुपाएँ 

```

Comment:
Use "व्यापार" instead of "ट्रेड" for the English word "Trade" to maintain consistency across the translation.